### PR TITLE
feature/70-api-endpoints-consts

### PR DESCRIPTION
**Pull request by rmakara**
**Changes:**
* Added methods to `endpoints.ts` for Shopware API.
* Added tests for `endpoints.ts`.

**Issue:** #70

### CODE REVIEW

File: packages/shopware-6-client/__tests__/endpoints.spec.ts [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+const sampleCustomerId = "8b67c1fbb718487db750651430023298";
+const sampleLineItemId = "042df8812942487bb52c9fc1e5b26e20";
+const sampleOrderId = "27356105cf9b4484b96143881c37bbcb";
+
+describe("endpoints", () => {
+  describe("PRODUCT_ENDPOINT", () => {
+    it("should return Shopware product endpoint", async () => {
+      const result = await PRODUCT_ENDPOINT();
+      expect(result).toEqual("/product");
+    });
```
Comment (patzick) [inline packages/shopware-6-client/__tests__/endpoints.spec.ts]: Endpoint methods does not return a promise :)
We shouldn't `await` for non-promise method

```ts
    it("should return Shopware product endpoint", () => {
      const result = getProductEndpoint();
      expect(result).toEqual("/product");
    });
```

File: packages/shopware-6-client/src/endpoints.ts [hunk 2]
```diff
@@ -1 +1,79 @@
-export const CATEGORY_ENDPOINT = "/category";
+// category
+
+export const CATEGORY_ENDPOINT = () => `/category`;
```
Comment (patzick) [inline packages/shopware-6-client/src/endpoints.ts]: yeah, in fact, the name suggests that this is a const, not method
les's change these to a more method-conventional name like `getXxxEndpoint`
```suggestion
export const getCategoryEndpoint = () => `/category`;
```

**Comment by rmakara** [comment]
@patzick fixes applied

**Comment by mkucmus** [comment]
Yeah! now it's clear and ready to use.

**Pull request merged**